### ISSUE TITLE
RBM not training correctly

### ISSUE THREAD

**Issue by TTitcombe** [TTitcombe/Python-Machine-Learning-Models#2]
Image displayed during training immediately all white, with no change.

Mean weight value drops to negative. 

May be due to learning rate, or bug in training code.
